include/utility/vpd_specific_utility.hpp

```diff
@@ -377,5 +377,24 @@ inline void getVpdDataInVector(const std::string& vpdFilePath,
         throw;
     }
 }
+
+/**
+ * @brief An API to get D-bus representation of given keyword name
+ *
+ * @param[in] i_keywordName - Keyword name.
+ *
+ * @return D-bus representation of keyword name.
+ */
+inline std::string getDbusKeywordName(const std::string& i_keywordName)
+{
+    // D-bus doesn't support "#". Replace "#" with "PD_" for those "#"
+    // prefixed keywords.
+    if (i_keywordName.at(0) == constants::POUND_KW)
+    {
+        return (std::string(constants::POUND_KW_PREFIX) +
+                i_keywordName.substr(1));
```

Comment:
Just confirming, is this the guideline that only 1 character of the keyword name should be taken for conversion.
Eg: Lets say if the keyword name is "#ABC" then we will represent as "PD_A"?

Reply:
yes the IPZ  template says that keywords will be of 2 alpha numeric characters only.

Reply:
ok